external/llvm-project/mlir/include/mlir/Dialect/AMDGPU/Utils/Chipset.h

```diff
@@ -19,7 +19,7 @@ struct Chipset {
   static FailureOr<Chipset> parse(StringRef name);
 
   bool isGfx940() const {
-    return majorVersion == 9 && minorVersion >= 0x40 && majorVersion < 0x50;
+    return majorVersion == 9 && minorVersion >= 0x40 && minorVersion < 0x50;
```

Comment:
Argh.  I need to fix that in my upstream pull request, too.

Reply:
I think you'll want to pull all of this into your upstream PR?